refactor/data/app/core/localstoragebackend.py

```diff
@@ -108,7 +109,7 @@ def put(
             return ("", "")
         try:
             abs_p = str(p.resolve(True))
-            object_name = f"{uuid.uuid4()}.pkl"
+            object_name = f"{uuid.uuid4()}.pkl" if not overwrite else object_name
```

Comment:
Do you think it'll be better to use `tempfile._get_candidate_names()` instead of `uuid` here?

Reply:
This doesn't work for me. `tempfile._get_candidate_names()` returns a `tempfile._RandomNameSequence object`, so

`f"{tempfile._get_candidate_names()}.pkl"` evaluates to `'<tempfile._RandomNameSequence object at 0x7f40363062b0>.pkl'
`.

The current code works.

Reply:
Yeah, you're right. I was just wondering if there was a more standard/specific way of generating file names without creating files but it seems there isn't. So, yeah, `uuid` should work fine.